### PR TITLE
Restapi auto storage/resource manager

### PR DESCRIPTION
**Pull request by fengzhou-msft**
fix missing changes for LargeFileSharesState and others

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by Juliehzl** [comment]
LGTM

File: sdk/storage/azure-mgmt-storage/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+**Features**
+
+- Model StorageAccountCreateParameters has a new parameter large_file_shares_state
+- Model StorageAccountUpdateParameters has a new parameter large_file_shares_state
+- Model StorageAccount has a new parameter large_file_shares_state
+
+4.1.0 (2019-09-27)
+++++++++++++++++++
+
+**Features**
```
Comment (zikalino) [inline sdk/storage/azure-mgmt-storage/HISTORY.rst]: 4.1.0 appears twice here

File: sdk/storage/azure-mgmt-storage/HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,24 @@
 
 Release History
 ===============
+4.2.0 (2019-10-07)
```
Comment (zikalino) [inline sdk/storage/azure-mgmt-storage/HISTORY.rst]: and usually we have empty line here

Comment (fengzhou-msft) [reply sdk/storage/azure-mgmt-storage/HISTORY.rst]: resolved

**Pull request merged**